web/book/src/SUMMARY.md

```diff
@@ -55,6 +55,13 @@
   - [F-strings](./language-features/f-strings.md)
   - [Target & Version](./language-features/target.md)
 
+- [Compiler](./compiler/README.md)
+
+  - [Reference implementation](./compiler/rust/README.md)
+    - [CLI](./compiler/rust/cli.md)
+  - [Compiler features](./compiler/compiler-features/README.md)
+    - [Target & Version](./compiler/compiler-features/target.md)
```

Comment:
These make sense, but the span is quite low — 5 items in 3 levels. What about moving this section down a bit, and integrating it with `Internals`? Or at least moving a page like "Compiler architecture" here?

(I don't feel that strongly though, feel free to merge as-is and we can iterate)